sharepoint2019-non-ha/vnet-new.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
remove this and just use an if() statement on the dhcp options in the other template